cpp/vcpkg.json

```diff
@@ -157,7 +159,9 @@
     { "name": "s2n", "version": "1.3.5" },
     { "name": "spdlog", "version": "1.11.0" },
     { "name": "wangle", "version-string": "2022.03.21.00" },
-    { "name": "zlib", "version": "1.2.13" }
+    { "name": "zlib", "version": "1.2.13" },
+    { "name": "azure-identity-cpp", "version": "1.3.0" },
+    { "name": "azure-storage-blobs-cpp", "version": "12.6.2" }
```

Comment:
It's not necessary to pin the versions of new dependencies. The existing ones are pinned because we haven't had time to test their latest versions.

Reply:
No. Versions are already managed by the baseline.

The reason for these version overrides is explained on line 76/78. Your change is not related to that reason.